File: Practices/time_of_day.py
#CC 7th getting the time of day

time = int(input("What is the time of day using military time?(00-24, hour only)\n"))

if time >= 20:
    print("Goodnight")
elif time <=20 and time >=18:
    print("Good evening") 
elif time <=17 and time >= 12:
    print("Good afternoon")
elif time <= 5:
    print("Go to bed, actually I'm a hypocrite continue on.")
else:
    print("Good moring")

File: c_programming/practices/time_of_day.c
//CC 7th time of day
#include <stdio.h>
//Codespaces never lets me  run my code, so I am reveiwing this and hoping it runs properly.
int main(void){
    int time;

    printf("What is the current military time:\n");
    scanf("%d", time);

    if(time >= 5){
        printf("Go to bed please\n");
    }else if(time <=12 && time>= 6){
        printf("Good morning\n");
    }else if(time >=12 && time<=18){
        printf("Good afternoon\n");
    }else if(time >=18 && time <=20){
        printf("Good evening");
    }else{
        printf("Good night")
    }

    return 0;
}